src/index.js

```diff
@@ -0,0 +1,121 @@
+const state = {
+  Temperature: 0,
+  city: 'Seattle',
+  sky: 'sunny',
+  latitude: 0,
+  longitude: 0,
+};
+
+const increaseTemp = (event) => {
+  const tempContainer = document.querySelector('#Temperature');
+  console.log('increaseTemp clicked');
+  state.Temperature = Math.round(state.Temperature + 1);
+  tempContainer.textContent = `${state.Temperature}`;
+  tempRange();
+};
+// hello this is a new branch!! 
+
+const decreaseTemp = (event) => {
+  const tempContainer = document.querySelector('#Temperature');
+  state.Temperature = Math.round(state.Temperature - 1);
+  tempContainer.textContent = `${state.Temperature}`;
+  tempRange();
+};
+
+const tempRange = () => {
+  const tempContainer = document.querySelector('#Temperature');
+  const landscapeContainer = document.querySelector('#landscape');
+  if (state.Temperature >= 80) {
+    tempContainer.style.color = 'red';
+    landscapeContainer.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.Temperature <= 79 && state.Temperature >= 70) {
+    tempContainer.style.color = 'orange';
+    landscapeContainer.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.Temperature <= 69 && state.Temperature >= 60) {
+    tempContainer.style.color = 'yellow';
+    landscapeContainer.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.Temperature <= 59 && state.Temperature >= 50) {
+    tempContainer.style.color = 'green';
+    landscapeContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.Temperature <= 49) {
+    tempContainer.style.color = 'teal';
+    landscapeContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const updateSky = (event) => {
+  const skyContainer = document.querySelector('#skySelect');
+  const skyBannerContainer = document.querySelector('#skyBanner');
+  state.sky = skyContainer.value;
+  if (state.sky === 'sunny') {
+    skyBannerContainer.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (state.sky === 'cloudy') {
+    skyBannerContainer.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (state.sky === 'rainy') {
+    skyBannerContainer.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (state.sky === 'snowy') {
+    skyBannerContainer.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+};
+
+const chooseCityName = (event) => {
+  const cityName = document.querySelector('#cityName');
+  const inputContainer = document.querySelector('#inputCityName');
+  state.city = inputContainer.value;
+  cityName.textContent = state.city;
+};
+
+const resetCityName = (event) => {
+  const cityName = document.querySelector('#cityName');
```

Comment:
Looks like we are repeating code on lines 62 and 69! Maybe we can create global variables that will hold elements like this that will be used more than once